test/clojure/core_test/set.cljc

```diff
@@ -0,0 +1,23 @@
+(ns clojure.core-test.set
+  (:require clojure.core
+            [clojure.test :as t :refer [deftest testing is are]]
+            [clojure.core-test.portability #?(:cljs :refer-macros :default :refer)  [when-var-exists]]))
+
+(when-var-exists clojure.core/set
+  (deftest test-set
```

Comment:
I think we should have a test which asserts that `set` removes duplicates.

Something like `(set [1 2 3 12 3])`.